src/examples/vds_example.py

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python
+
+# Setup the name space:
+import requests
+
+# Start by making a request to the root URI of the VDS API:
+r0 = requests.get("http://odin.rss.chalmers.se/rest_api/v4/vds/")
+
+# The request contains the returned JSON object, which in Python is a
+# dictionary.  This can be printed or inspected to find out its keys and
+# contents.  Let's assume that we have done that, or that we have read
```

Comment:
multiple spaces in comment

Reply:
This is American standard, but I suppose we are all Brits...